site/en/docs/extensions/reference/scripting/index.md

```diff
@@ -2,9 +2,191 @@
 api: scripting
 ---
 
+## Manifest
+
+In order to use the `chrome.scripting` API, you need to specify a
+`"manifest_version"` of `3` or higher and include the `"scripting"` permission
+in your [manifest file](manifest).
+
+```json
+{
+  "name": "Scripting Extension",
+  "manifest_version": 3,
+  "permissions": ["scripting"],
+  ...
+}
+```
+
+## Usage
+
+You can use the `chrome.scripting` API to inject JavaScript and CSS into
+websites, similar to what can be achieved with
+[content scripts](contentScripts). With the `chrome.scripting` API, extensions
+can make decisions at runtime.
```

Comment:
Suggestion:
You can use the `chrome.scripting` API to inject JavaScript and CSS into
websites. This is similar to what you can do with
[content scripts](contentScripts), but by using the `chrome.scripting` API, extensions can make decisions at runtime.

Reply:
Done